Rohit/PostgresAssignment/controller/class.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "vm";
+
+const cls = require('../sql/class');
+
+interface classesinfo {
+    cls_id: number;
+    st_id: number;
+  }
+  
+  
+exports.classData = async (ctx: Context) => {
+    try {
+      let [rows]: Array<{ rows: classesinfo }> = [];
+      rows = await cls.get_classes();
+  
+      ctx.response.status = 200;
+      ctx.response.type = "application/json";
+      ctx.body = rows.rows;
+    } catch (err) {
+      ctx.response.status = 500;
```

Comment:
can you check what type of error it is if it a db error we can show some different error message. Like trying to update a record but dosnt have match row, db will throw no recod found or something. this give user better error and they can fix the issue. Check with Druval once on this.

Reply:
do this in all the places